src/components/application_manager/include/application_manager/smart_object_keys.h

```diff
@@ -280,6 +280,7 @@ const char location_name[] = "locationName";
 const char location_description[] = "locationDescription";
 const char address_lines[] = "addressLines";
 const char phone_number[] = "phoneNumber";
+const char number[] = "number";
```

Comment:
@AByzhynar 
As I found out 'phone_number' is being used in SendLocation request according to its API.
New field 'number' is necessary for DialNumber request.